server/routes/office.js

```diff
@@ -9,7 +9,7 @@ const officeRoute = express.Router();
 officeRoute.post('/create', validate, office.officeTable);
 officeRoute.post('/', validate, office.create);
 // officeRoute.get('/', office.getAllOffices);
-// officeRoute.get('/:id', office.getOneOffice);
+officeRoute.get('/:id/result', office.results);
 // officeRoute.delete('/:id', office.deleteOne);
```

Comment:
here too

Reply:
marge after apply feedback

Reply:
marge after apply feedback
```suggestion
// officeRoute.delete('/:id', office.deleteOne);
```